src/modules/powerrename/UWPui/PowerRenameUWPUI.cpp

```diff
@@ -4,6 +4,7 @@
 #include <CLSID.h>
 #include <PowerRenameExt.h>
 #include <common.h>
+#include <com_object_factory.h>
```

Comment:
Let's revert this change to PowerRenameUWPUI.cpp and open an issue to track it.

Reply:
https://github.com/microsoft/PowerToys/issues/1367